### PR TITLE
WheelPicker - add color to labelStyle

### PR DESCRIPTION
**Pull request by M-i-k-e-l**

### CODE REVIEW

File: lib/components/WheelPicker/index.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ class WheelPicker extends Component {
      */
     style: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
     /**
-     * pass custom label style: fontSize, fontFamily
+     * pass custom label style: fontSize, fontFamily, color<br>
+     * Note: label's color will override the WheelPicker's color<br>
```
Comment (Inbal-Tish) [inline lib/components/WheelPicker/index.js]: Why is the label color overrides the picker color?

File: lib/components/WheelPicker/index.js [hunk 2]
```diff
@@ -91,6 +93,11 @@ class WheelPicker extends Component {
   render() {
     const {style, color, labelStyle, itemHeight} = this.props;
     const {fontSize, fontFamily} = labelStyle;
+    let {color: labelColor} = labelStyle;
```
Comment (Inbal-Tish) [inline lib/components/WheelPicker/index.js]: What is `labelColor` here?

File: lib/components/WheelPicker/index.js (hunk 1 above)
Comment (M-i-k-e-l) [reply lib/components/WheelPicker/index.js]: That was the requirement as I understood it

File: lib/components/WheelPicker/index.js (hunk 2 above)
Comment (M-i-k-e-l) [reply lib/components/WheelPicker/index.js]: The color of the text

**Pull request merged**